src/components/List_Products.js

```diff
@@ -0,0 +1,52 @@
+import React, { useEffect, useState } from 'react';
+import { saveItem } from '../lib/api';
+import { db } from '../lib/firebase';
+import { collection, onSnapshot } from 'firebase/firestore';
+
+function ListProducts() {
+  const [items, setItems] = useState(null);
+  const [itemName, setItemName] = useState('');
+
+  /* Get items */
+  useEffect(() => {
+    const unsubscribe = onSnapshot(collection(db, 'Products'), (snapshot) => {
+      let products = [];
+
+      snapshot.forEach((doc) => {
+        products.push({ ...doc.data(), id: doc.id });
+      });
+
+      console.log(products);
+      setItems(products);
+
+      return () => {
+        unsubscribe();
+      };
+    });
+  }, []);
+
+  /* Save Item */
+  const saveItemName = async (e) => {
+    e.preventDefault();
+    saveItem(itemName);
+    setItemName('');
+  };
```

Comment:
The `saveItemName` function is an interesting case about the `async/await` statements. On another hand, it's ok that you might use **async functions** without `await` (in some cases). But, in this specific case, the `saveItem` might return a promise or not. Then, if it returns a promise, you should consider [the `await` keyword](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function). (e.g. `await saveItem(itemName)`)